_config.yml

```diff
@@ -15,12 +15,26 @@ exclude:
 - work
 - ".sass-cache"
 
+# defaults
+defaults:
+  -
+    scope:
+      path: ""
+      type: agencies
+    values:
+      layout: default
+
+collections:
+  agencies:
+    output: true
+    permalink: /:path/
```

Comment:
In our previous work, we used `/agencies/:path/`. I'm up for either one, I just want to make sure this is deliberate.